crates/model/src/signature.rs

```diff
@@ -32,12 +33,28 @@ impl From<QuoteSigningScheme> for SigningScheme {
     }
 }
 
+/// Signature over the EIP-712 hash of the order data. This hash is referred to as the order hash.
 #[derive(Eq, PartialEq, Clone, Deserialize, Serialize, Hash)]
 #[serde(into = "JsonSignature", try_from = "JsonSignature")]
 pub enum Signature {
+    /// The order hash is signed according to EIP-712.
```

Comment:
Somewhat nitpick: technically the signature enum `Signature` is for the order data, the order hash is only an implementation detail. 

```suggestion
/// Signature over the order data.
/// All variants rely on the EIP-712 hash of the order data, referred to as the order hash.
#[derive(Eq, PartialEq, Clone, Deserialize, Serialize, Hash)]
#[serde(into = "JsonSignature", try_from = "JsonSignature")]
pub enum Signature {
    /// The order struct is signed according to EIP-712.
```